Movecraft/src/main/java/net/countercraft/movecraft/async/translation/TranslationTask.java

```diff
@@ -230,12 +230,19 @@ protected void execute() throws InterruptedException, ExecutionException {
                     newHitBox.getMidPoint().getZ());
             test = test.translate(0, -1, 0);
             while (test.toBukkit(world).getBlock().getType().isAir()) {
+                // If we are out of bounds, that is technically air, but can we hover over ground???
+                if (test.getY() < world.getMinHeight()) {
+                    fail(String.format(I18nSupport.getInternationalisedString("Translation - Failed Craft over void")));
+                    return;
```

Comment:
We should break out of the loop here, it's forbidden hover over blocks not forbidden hover over void (perhaps add that as another option in another PR?).

Reply:
Message needs to be changed. And as of now, the relevant code here causes a infinite loop when the vehicle goes over void.

Reply:
The while condition could also just be changed to <current condition> && positionIsInWorld(), should also fix that.

Reply:
Changed the loop condition for now. Hover-over-void is a good idea for a separate PR and separate option, will add that to my todo list